drivers/bluetooth/btintel.c

```diff
@@ -3061,6 +3061,7 @@ static int btintel_setup_combined(struct hci_dev *hdev)
 	}
 
 	/* Apply the common HCI quirks for Intel device */
+        set_bit(HCI_QUIRK_RESET_ON_CLOSE, &hdev->quirks);
```

Comment:
is this change specific to BT SCO or other issue?

Reply:
This change fixes below issue.

Bluetooth: btusb: Add reset on close quirk for Intel adapters

Intel Bluetooth adapter is not exiting the loopback mode on bluetooth
socket closure. So, when bluetooth socket is opened again, bluetooth is
not working.

As per Bluetooth core specification, bluetooth adapter should exit loop
back mode on hci reset. Add HCI quirk reset to make sure adapter exits
loopback mode on bluetooth socket closure.